staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/README.md

```diff
@@ -0,0 +1,4 @@
+This directory contains a port of https://github.com/google/cel-policy-templates-go/tree/master/policy/model modified in a few ways:
+
```

Comment:
is this expected to be refreshed from `https://github.com/google/cel-policy-templates-go/tree/master/policy/model`? if so, should the deltas be made in their own commits so they can be replayed on top of refreshed copies re-imported later?

Reply:
Good call. I've split out the delta.